app/assets/stylesheets/donor.css.scss

```diff
@@ -236,3 +236,12 @@
     font-weight: 500;
   }
 }
+
+.best_in_place:hover {
+    cursor: pointer;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered background-color, border-radius, color, cursor, padding-left, padding-right